lib/Mojolicious/Plugin/OpenAPI/Guides/OpenAPI3.pod

```diff
@@ -0,0 +1,392 @@
+=head1 NAME
+
+Mojolicious::Plugin::OpenAPI::Guides::OpenAPI3 - Mojolicious <3 Open API (Swagger)
+
+=head1 OVERVIEW
+
+This guide will give you an introduction to how to use
+L<Mojolicious::Plugin::OpenAPI>.
+
+=head1 TUTORIAL
+
+=head2 Specification
+
+This plugin reads an L<OpenAPI specification|https://openapis.org/specification>
+and generate routes and input/output rules from it. See L<JSON::Validator> for
+L<supported schema file formats|JSON::Validator/Supported schema formats>.
+
+  {
+    "openapi": "3.0.2",
+    "info": {
+      "version": "1.0",
+      "title": "Some awesome API"
+    },
+    "paths": {
+      "/pets": {
+        "get": {
+          "operationId": "getPets",
+          "x-mojo-name": "get_pets",
+          "x-mojo-to": "pet#list",
+          "summary": "Finds pets in the system",
+          "parameters": [
+            {
+              "in": "query",
+              "name": "age",
+              "schema": {
+                "type": "integer"
+              }
+            }
```

Comment:
This is something I'm very unsure about, but should some of these objects be on one line? Would it make it easier or harder to read?

Reply:
I think it would make it harder to read, on the other hand, it takes up a lot of space as well

Reply:
I might change it later if I think it makes it easier to read, but gonna leave it as is for now.